include/IMesh.h

```diff
@@ -92,10 +95,18 @@ namespace scene
 	class ICPUMesh : public IMesh<scene::ICPUMeshBuffer>, public core::BlobSerializable
 	{
 	public:
+		//! Serializes mesh to blob for *.baw file format.
+		/** @param _stackPtr Optional pointer to stack memory to write blob on. If _stackPtr==NULL, sufficient amount of memory will be allocated.
+			@param _stackSize Size of stack memory pointed by _stackPtr.
+			@returns Pointer to memory on which blob was written.
+		*/
 		virtual void* serializeToBlob(void* _stackPtr = NULL, const size_t& _stackSize = 0) const
 		{
 			return core::CorrespondingBlobTypeFor<ICPUMesh>::type::createAndTryOnStack(this, _stackPtr, _stackSize);
 		}
+
+		//! Adds a mesh buffer.
+		virtual void addMeshBuffer(ICPUMeshBuffer* buf) = 0;
```

Comment:
There's an SGPUMesh for such operations on an IGPUMesh

should there be an SCPUMesh derived from ICPUMesh too?
(if one wants to add and remove meshbuffers)